test/test_helper.js

```diff
@@ -0,0 +1,4 @@
+import chai from 'chai';
```

Comment:
chai api is pretty straightforward. this is the tutorial i'm working off of to update the way we're using react components and eventually to setup up redux / immutablejs.
http://teropa.info/blog/2015/09/10/full-stack-redux-tutorial.html